test/studies/isx/PERFEXECOPTS

```diff
@@ -1 +1 @@
---mode=weakISO --useSubTimers=true
+--mode=weakISO --useSubTimers=true --n 8388608
```

Comment:
For simplicity/fewer magic numbers, could this have used -perBucketMultiply=1 as well?